src/foam/nanos/crunch/CapabilityJunctionStatus.js

```diff
@@ -11,61 +11,61 @@ foam.ENUM({
       name: 'PENDING',
       label: { en: 'pending', pt: 'pendente'},
       documentation: 'Our team is currently reviewing. Approvals may take up to 24 hours',
-      background: '#E7EAEC',
-      color: '#1E1F21'
+      background: '#FFFFFF',
```

Comment:
Is there anyway to have these use the theme colours? Like what was used in the ApprovalStatus

Reply:
These Status indicators don't use the theme colours so we kind of have to hard code them.

Reply:
Could be something we could discuss with the design team regarding colour consistency. I believe it is done like this to ensure legibility over difference colour backgrounds. For now I'll leave it at this and we can come back to it.

Reply:
Then open a new ticket so we don't forget.